AngularApp/src/app/shared/models/detectorCommit.ts

```diff
@@ -0,0 +1,6 @@
+export interface DetectorCommit {
```

Comment:
Lets be consistent with lowercase file names, so change this to detector-commit.ts